src/Raven.Server/Documents/Sharding/ShardedRequestHandler.cs

```diff
@@ -1,16 +1,16 @@
-﻿using System;
+﻿using System.Text;
 using Raven.Server.ServerWide.Context;
 using Raven.Server.Web;
 using Sparrow;
-using Sparrow.Json;
 
-namespace Raven.Server.Documents
+namespace Raven.Server.Documents.Sharding
 {
     public class ShardedRequestHandler : RequestHandler
     {
         public ShardedContext ShardedContext;
         public TransactionContextPool ContextPool;
 
+        protected static readonly ObjectPool<StringBuilder> StringBuilderObjectPool = new ObjectPool<StringBuilder>(()=> new StringBuilder());
```

Comment:
unused

Reply:
yes I thought of using object pools for the managed objects I need but decided that a private member will be better